content/docs/command-reference/exp/index.md

```diff
@@ -39,16 +41,17 @@ positional arguments:
 `dvc exp` subcommands provide specialized ways to create and manage data science
 experiments. The basic workflow goes like this:
 
-- Modify <abbr>dependencies</abbr> (e.g. input data or source code),
-  <abbr>parameters</abbr>, or commands (`cmd` field of `dvc.yaml`) of committed
-  stages.
-- Use `dvc exp run` (instead of `repro`) to execute the pipeline. This puts the
-  experiment's results in your <abbr>workspace</abbr>, and tracks it under the
-  hood.
-- Visualize experiment configurations and results with `dvc exp show`. Repeat.
+- Modify stage <abbr>parameters</abbr> or other dependencies (e.g. input data,
+  source code) of committed stages.
+- Use `dvc exp run` (instead of `repro`) to execute the pipeline. The results
```

Comment:
This list is repeated from content/docs/user-guide/experiment-management.md which may not be the best idea... For now should work — not sure what else to put here (maybe nothing).